Make.config

```diff
@@ -189,6 +189,12 @@ MIN_WATCH_OS_VERSION=2.0
 MIN_TVOS_SDK_VERSION=9.0
 MIN_MACCATALYST_SDK_VERSION=13.1
 
+# Minimum OS versions for dotnet6 support
```

Comment:
I wonder if we should have these inside the (Make.versions)[https://github.com/xamarin/xamarin-macios/blob/main/Make.versions] file. Just a nit really and possibly out of the scope of this PR

```shell
# Minimum OS versions for running XI/XM apps.
MIN_IOS_SDK_VERSION=7.0
MIN_OSX_SDK_VERSION=10.9
MIN_WATCHOS_SDK_VERSION=2.0
MIN_WATCHOS64_32_SDK_VERSION=5.1
MIN_WATCH_OS_VERSION=2.0
MIN_TVOS_SDK_VERSION=9.0
MIN_MACCATALYST_SDK_VERSION=13.1

# Minimum OS versions for dotnet6 support
DOTNET_MIN_IOS_SDK_VERSION=10.0
DOTNET_MIN_TVOS_SDK_VERSION=10.0
DOTNET_MIN_MACCATALYST_SDK_VERSION=13.1
DOTNET_MIN_OSX_SDK_VERSION=10.14
```